samples/TagHelperSample.Web/TagHelpers/ConditionalCommentTagHelper.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Razor.Runtime.TagHelpers;
+
+namespace TagHelperSample.Web
+{
+    [TargetElement("iecondition")]
+    public class ConditionalCommentTagHelper : TagHelper
+    {
+        [HtmlAttributeName("mode")]
+        public string Mode { get; set; }
+
+        [HtmlAttributeName("condition")]
+        public string Condition { get; set; }
+
+        public override void Process(TagHelperContext context, TagHelperOutput output)
+        {
+            output.TagName = null;
+            string startCondition = "";
+            string endCondition = "";
+            if (Mode == "downlevelHidden")
+            {
+                startCondition = $"<!--[if {Condition}]>";
+                endCondition = "<![endif]-->";
+
+            }
+            else if (Mode == "downlevelRevealed")
+            {
+                startCondition = $"<![if {Condition}]>";
```

Comment:
What exactly are the encoding requirements for an IE conditional comment? This could have problems with HTML encoding. If so, much better to explicitly `Append()` the different parts because they have distinct encoding requirements e.g.
``` c#
output.PreContent.Append("<![if ");
output.PreContent.Append(new StringHtmlContent(Condition));
output.PreConent.Append("]>");
```

If HTML encoding were not an issue, should still be done using simple string concatenation.